### PR TITLE
error cleanup

### PR DESCRIPTION
**Pull request by brandhayze**

### CODE REVIEW

File: client/components/SuggestionsList.tsx [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+interface Suggestion {
+  phrase: string;
+  group: string;
+}
+
+interface SuggestionsListProps {
+  suggestions: Suggestion[];
+}
+
+const SuggestionsList: React.FC<SuggestionsListProps> = ({ suggestions }) => {
```
Comment (n1ce1041) [inline client/components/SuggestionsList.tsx]: Any chance we can amend this to https://stackoverflow.com/questions/59988667/typescript-react-fcprops-confusion

See discussion from CRA   team:

https://github.com/facebook/create-react-app/pull/8177

**Review by n1ce1041 — CHANGES_REQUESTED** [review]
Amend React.FC to the 

`const PrintName2 = ({ prop1, prop2 }: Props): JSX.Element => { /** */}
`

Format pls

File: client/components/SuggestionsList.tsx (hunk 1 above)
Comment (brandhayze) [reply client/components/SuggestionsList.tsx]: On it boss

**Review by n1ce1041 — APPROVED** [review]
Looks good. Mergify this bizzatch into MASTER. 

https://images2.memedroid.com/images/UPLOADED47/529c0362e0710.jpeg

**Pull request merged**